pegasus/sites.v3/code.org/public/css/brstrap.css

```diff
@@ -62,8 +68,44 @@
   .mobile-show-medium { display: none }
 
   .i-agree-button { width: 9%; }
-}
 
+ /*This is applicable when side by side images with rounded corners need to be stacked on smaller sizes (e.g. code.org/student)*/
```

Comment:
brstrap was designed to be content agnostic, without specific types of content being mentioned.  (`.i-agree-button` is a clear exception.)  Can we find a way to continue keeping brstrap generic to the content layouts?